static/app/components/events/breadcrumbs/breadcrumbsDrawerContent.tsx

```diff
@@ -0,0 +1,241 @@
+  const [sort, setSort] = useLocalStorageState<BreadcrumbSort>(
+    BREADCRUMB_SORT_LOCALSTORAGE_KEY,
+    BreadcrumbSort.NEWEST
+  );
+  const [timeDisplay, setTimeDisplay] = useLocalStorageState<BreadcrumbTimeDisplay>(
+    BREADCRUMB_TIME_DISPLAY_LOCALSTORAGE_KEY,
+    BreadcrumbTimeDisplay.RELATIVE
+  );
+  const filterOptions = useMemo(
+    () => getBreadcrumbFilters(allBreadcrumbs),
+    [allBreadcrumbs]
+  );
+
+  const displayCrumbs = useMemo(() => {
+    const sortedCrumbs =
+      sort === BreadcrumbSort.OLDEST ? allBreadcrumbs : [...allBreadcrumbs].reverse();
+    const filteredCrumbs = sortedCrumbs.filter(bc =>
+      filterSet.size === 0 ? true : filterSet.has(bc.type)
+    );
+    const searchedCrumbs = applyBreadcrumbSearch(search, filteredCrumbs);
+    return searchedCrumbs;
+  }, [allBreadcrumbs, sort, filterSet, search]);
+
+  const startTimeString = useMemo(
+    () =>
+      timeDisplay === BreadcrumbTimeDisplay.RELATIVE
+        ? displayCrumbs?.at(0)?.timestamp
+        : undefined,
+    [displayCrumbs, timeDisplay]
+  );
+
+  const actions = (
+    <ButtonBar gap={1}>
+      <InputGroup>
+        <SearchInput
+          size="xs"
+          value={search}
+          onChange={e => setSearch(e.target.value)}
+          autoFocus={focusControl === BreadcrumbControlOptions.SEARCH}
+        />
+        <InputGroup.TrailingItems disablePointerEvents>
+          <IconSearch size="xs" />
+        </InputGroup.TrailingItems>
+      </InputGroup>
+      <CompactSelect
+        size="xs"
+        onChange={options => {
+          const newFilters = options.map(({value}) => value);
+          setFilterSet(new Set(newFilters));
+        }}
+        multiple
+        options={filterOptions}
+        maxMenuHeight={400}
+        trigger={(props, isOpen) => (
+          <BreadcrumbControl
+            size="xs"
+            isActive={filterSet.size > 0}
+            icon={<IconFilter />}
+            showChevron={false}
+            isOpen={isOpen}
+            autoFocus={focusControl === BreadcrumbControlOptions.FILTER}
+            {...props}
+          >
+            {filterSet.size > 0 ? filterSet.size : null}
+          </BreadcrumbControl>
+        )}
+      />
+      <CompactSelect
+        size="xs"
+        trigger={(props, isOpen) => (
+          <BreadcrumbControl
+            size="xs"
+            icon={<IconSort />}
+            showChevron={false}
+            isOpen={isOpen}
+            autoFocus={focusControl === BreadcrumbControlOptions.SORT}
+            {...props}
+          />
+        )}
+        onChange={selectedOption => setSort(selectedOption.value)}
+        value={sort}
+        options={BREADCRUMB_SORT_OPTIONS}
+      />
+      <CompactSelect
+        size="xs"
+        trigger={(props, isOpen) => (
+          <BreadcrumbControl
+            size="xs"
+            icon={<IconClock />}
+            showChevron={false}
+            isOpen={isOpen}
+            {...props}
+          />
+        )}
```

Comment:
i think you might be able to do this with triggerProps `triggerProps={{icon:  <IconClock />, borderless: true}}` or at least i think get rid of the styled component with borderless
```suggestion
        trigger={(props, isOpen) => (
          <BreadcrumbControl
            size="xs"
            icon={<IconClock />}
            showChevron={false}
            isOpen={isOpen}
            {...props}
          />
        )}
```